### ISSUE TITLE
vstor_redist.exe not download

### ISSUE THREAD

**Issue by yythoss** [bovender/VstoAddinInstaller#11]
I just installed a fresh Windows 10 32Bit and started my setup. However, vstor_redist.exe was not downloaded.
What could that be?

**Comment by bovender** [comment]
Do you get to see any error messages? What's in the install log?

The hard-coded URL (in `defines.iss`) for the VSTOR redistributable still works for me when I point my browser to it:

http://download.microsoft.com/download/7/A/F/7AFA5695-2B52-44AA-9A2D-FC431C231EDC/vstor_redist.exe

Are you able to access this URL?

**Comment by yythoss** [comment]
Yes this hard-coded link works. But the Logfile show VSTO not needed:

2021-01-07 13:02:07.182   NeedToInstallVstor: Minimum required VSTOR 2010 build: 40305
2021-01-07 13:02:07.182   NeedToInstallVstor: No need to install VSTO runtime

**Comment by yythoss** [comment]
I see this function:

}
function NeedToInstallVstor(): boolean;
begin
  Log('NeedToInstallVstor: Minimum required VSTOR 2010 build: ' + IntToStr(MIN_VSTOR_BUILD));
  result := false; { Default }
  if IsOffice2007Installed or IsOffice2010Installed then
    result := (GetVstorBuild < MIN_VSTOR_BUILD) and not IsUpdate;
  if result then
    Log('NeedToInstallVstor: Need to install VSTO runtime')
  else
    Log('NeedToInstallVstor: No need to install VSTO runtime');
end;

On the Windows 10 Computer is Office 2013 installed, but not vstor_redist.exe.
So the addin not work.

**Comment by yythoss** [comment]
PrerequisitesAreMet return also true, so never check the rest.

**Comment by yythoss** [comment]
Now I change the Script and the download work, but no install Page come up.
Do you tested your Software today on a Windows Maschine?

CreateInstallInfoPage was called but not show.
